lib/ssh_scan/database/database_config.yml

```diff
@@ -0,0 +1,20 @@
+# MongoDB Example
+database:
+  type: mongodb
+  name: ssh_scan
+  server: 127.0.0.1
+  port: 27017
```

Comment:
As discussed in the MWoS meeting today, we should have SQLite as the default to make it easier for new users to stand up a basic implementation and if later they want to switch to mongo, all they need to do is change the config and restart their API server.